ui/app/(ai)/analyst/prompts.ts

```diff
@@ -0,0 +1,381 @@
+const supervisorPrompt = `
```

Comment:
Following up on this comment: https://github.com/prowler-cloud/prowler/pull/7508#discussion_r2052086824 

Could we please use the `/app` folder only for routing purposes?

Thank you!

Reply:
Moved code to `/lib/lighthouse/prompts.ts`